src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final String DELIMITER =",";
+    public static final String OPERATION ="buy";
+    int supplyAmount = 0;
+    int buyAmount = 0;
+    int result = 0;
+    String operation;
+    String amount;
+    List<String> fileData = null;
```

Comment:
The variables 'operation', 'amount', and 'fileData' should be declared inside the method where they are used. There is no need to have them as class fields since they don't represent the state of the object.